modules/exploits/linux/local/service_persistence.rb

```diff
@@ -77,6 +78,11 @@ def initialize(info = {})
         OptString.new('SERVICE', [false, 'Name of service to create'])
       ], self.class
     )
+    register_advanced_options(
+      [
+          OptBool.new('Enable', [true, 'Enable the service', true])
+      ], self.class
```

Comment:
`, self.class` should not be necessary.

```suggestion
      ]
```